autom8s/helm.js

```diff
@@ -48,18 +52,30 @@ class Helm {
 
   async delete(delOptions) {
     const { releaseName } = delOptions;
-    console.log(`deleting release: ${releaseName}`);
+    Helm._verifyNotEmpty(releaseName, 'releaseName');
 
+    console.log(`deleting release: ${releaseName}`);
     return this._executeHelm(`delete ${releaseName}`);
   }
 
   async upgrade(deployOptions) {
     const chartName = deployOptions.chartName.toLowerCase();
     const releaseName = deployOptions.releaseName.toLowerCase();
+
+    Helm._verifyNotEmpty(chartName, 'chartName');
+    Helm._verifyNotEmpty(releaseName, 'releaseName');
+
     const upgradeCommand = `upgrade ${releaseName} ${chartName}`;
-    console.log(`upgradeCommand command: ${upgradeCommand}`);
+    console.log(`upgrade command: ${upgradeCommand}`);
+    return this._installOrUpgradeChart(upgradeCommand, deployOptions);
+  }
 
-    return this._innerInstallUpgrade(upgradeCommand, deployOptions);
+  static _verifyNotEmpty(arg, argName) {
+    if (typeof arg === 'undefined' || arg === null || arg === '') {
+      const errorMsg = `${argName} is mandatory`;
```

Comment:
change 'mandatory' to 'required' whch is more conventional (e.g. required or optional parameters)